base/src/main/java/io/fabric8/launcher/base/http/HttpClient.java

```diff
@@ -157,19 +162,42 @@ public void onResponse(Call call, Response response) {
         }
     }
 
-    private static final TrustManager[] trustAllCerts = {
-            new X509TrustManager() {
+    public static final TrustManager[] trustAllCerts = {
```

Comment:
Not sure I like this. I mean it's a static array being exposed as a public attribute :)